recipes-graphics/imx-gpu-viv/imx-gpu-viv-6.inc

```diff
@@ -61,6 +61,10 @@ REQUIRED_DISTRO_FEATURES_mx8 = "wayland"
 
 SRC_URI = "${FSL_MIRROR}/${BPN}-${PV}.bin;fsl-eula=true"
 
+PACKAGECONFIG ?= ""
+
+PACKAGECONFIG[valgrind] = ""
```

Comment:
Please add a comment on top of the `PACKAGECONFIG` entry explaining what does it provides.